docs/configure/overview.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 'Overview'
+---
+
+Storybook is configured via a folder, called `.storybook` which contains various configuration files. 
+
+> Note you can change the folder that Storybook uses by setting the `-c` flag to your `start-storybook` and `build-storybook` scripts.
+
+
+### Configure your Storybook project
+
+The main configuration file is `main.js`. This file controls the behaviour of the Storybook server, and so you must restart Storybook’s process when you change it. It contains the following:
+
+```js
+// .storybook/main.js
+
+module.exports = {
+  stories: ['../src/**/*.stories.(js|mdx)'],
+  addons: ['@storybook/addon-essentials']
+}
+```
+
+The `main.js` configuration file is a [preset](../presets/introduction) and as such has a powerful interface, but the key fields within it are:
+
+- `stories` - a glob that indicates the [location of your story files](locate-story-files), relative to `main.js`.
+- `addons` - a list of the [addons](/addons) you are using.
+- `webpackFinal` - custom [webpack configuration](./integration#extending-storybooks-webpack-config).
+- `babel` - custom [babel configuration](integration#babel).
+
+### Configure story loading
+
+By default Storybook will load stories from your project based on a glob (pattern matching string) in `.storybook/main.js`. By default that matches all files in your project with extension `.stories.js` -- the intention is you would colocate those files with the component they document.
+
+```
+•
+└── components
+        ├── Button.js
+        └── Button.stories.js
+```
+
+If you want to use a different naming convention, you can alter the glob, using the syntax supported by [micromatch](https://github.com/micromatch/micromatch#extended-globbing).
```

Comment:
Maybe we should add an example here.

Reply:
if you can spare me one. I can gladly add it.

Reply:
Hmm, something like: 

> For example, if you wanted to pull `.md` and `.js` files from the `design-system/src/components` directory, you could write:
> 
> ```
> // .storybook/main.js
> module.exports = {
>   stories: ['../design-system/src/components/*.@{js,md}'],
> };
```

(can you double check my syntax @jonniebigodes? thx)

Reply:
I've changed it and included the example. According to [this](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md) you were almost right...

I think the correct way to use should be 

```js
module.exports = {
  stories: ['../design-system/src/components/*.@(js|md)'],
};
```

Feel free to correct me if i'm wrong.